task-maker-async/src/store.rs

```diff
@@ -364,6 +367,12 @@ impl StoreService {
         let mut service = self.service.lock().unwrap();
         let hash = FileSetHash(computation, variant);
         let handle = service.create_or_open_file_set(hash, FileSetKind::Computation)?;
+        // Notify waiters.
+        if let Some(x) = service.waiting_computation_readers.get_mut(&hash) {
+            x.drain(..).for_each(|waiter| {
+                waiter.send(()).unwrap();
```

Comment:
Is it possible that the receiver gets dropped and this panics?

Reply:
yup, fixed